src/test/java/com/iota/iri/IXITest.java

```diff
@@ -1,79 +1,80 @@
     static IXI ixi;
 
+    /**
+     * Create IXI temporary directory and start IXI.
+     * @throws Exception if temporary folder can not be created.
+     */
     @BeforeClass
     public static void setUp() throws Exception {
         ixiDir.create();
         ixi = new IXI();
-        ixi.init(ixiDir.getRoot().getAbsolutePath().toString());
+        ixi.init(ixiDir.getRoot().getAbsolutePath());
     }
 
+    /**
+     * Shutdown IXI and delete temporary folder.
+     * @throws InterruptedException if directory watch thread was interrupted.
+     */
     @AfterClass
-    public static void tearDown() throws Exception {
+    public static void tearDown() throws InterruptedException {
         ixi.shutdown();
         ixiDir.delete();
     }
 
+    /**
+     * If an command matches the command pattern, but is not valid, expect an unknown command error message.
+     */
     @Test
-    public void init() throws Exception {
-        AbstractResponse response;
-        IXIResponse ixiResponse;
-
-        /*
-        final String testJs =
-                "var Callable = Java.type(\"com.iota.iri.service.CallableRequest\");\n" +
-                        "print(\"hello world\");\n" +
-                        "var IXIResponse = Java.type(\"com.iota.iri.service.dto.IXIResponse\");\n" +
-                        "API.put(\"getParser\", new Callable({\n" +
-                        "call: function(req) {\n" +
-                        "var IntArray = Java.type(\"int[]\");\n" +
-                        "var out = new IntArray(Math.floor(Math.random()*9)+1);\n" +
-                        "out[0] = 2;\n" +
-                        "var r = IXIResponse.create({\n" +
-                        "myArray: out,\n" +
-                        "name: \"Foo\"\n" +
-                        "});\n" +
-                        "return r;\n" +
-                        "}\n" +
-                        "}));";
-        final String testPackage = "{\"main\": \"index.js\"}";
-
-
-        ixiDir.newFolder("test");
-        try (OutputStream out = new BufferedOutputStream(
-                Files.newOutputStream(Paths.get(ixiDir.getRoot().toString(),"test", "index.js"), CREATE))) {
-            out.write(testJs.getBytes());
-        }
-        try (OutputStream out = new BufferedOutputStream(
-                Files.newOutputStream(Paths.get(ixiDir.getRoot().toString(),"test", "package.json"), CREATE))) {
-            out.write(testPackage.getBytes());
-        }
-        // Allow IXI to load the file
-        Map<String, Object> request = new HashMap<>();
-        Thread.sleep(1000);
-        response = IXI.instance().processCommand("test.getParser", request);
-
-        assertFalse(response instanceof ErrorResponse);
-        assertTrue(response instanceof IXIResponse);
+    public void processCommandError() {
+        AbstractResponse response = ixi.processCommand("testCommand.testSuffix", null);
+        assertNotNull("Response should not be null", response);
```

Comment:
I am not sure if it makes sense to force comments on method level and assertion level. The tests are very small and sometimes documentation is not necessary.

Reply:
I will turn this off for tests